src/metabase/api/routes.clj

```diff
@@ -87,6 +89,9 @@
   (context "/slack"                [] (+auth slack/routes))
   (context "/table"                [] (+auth table/routes))
   (context "/task"                 [] (+auth task/routes))
+  (context "/testing"              [] (if (env/env :mb-enable-test-endpoints)
```

Comment:
Don't you want to rather use `(metabase.config/config-bool :mb-enable-test-endpoints)` here? Using `env/env` directly means that this will (unintentionally) enable the testing routes when `MB_ENABLE_TEST_ENDPOINTS` is any one of `0`, `false` or empty. It will _only_ evaluate to `false` here if that env var is not set.

Reply:
Presumably `MB_ENABLE_TEST_ENDPOINTS` is never set unless you want to enable it, but I'll go ahead change it.